### PR TITLE
Test the add attachment button

### PR DESCRIPTION
**Pull request by zmbc**

### CODE REVIEW

File: src/base/static/components/form-fields/types/__tests__/add-attachment-button.test.js [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    expect(toDataUrlCalls.length).toEqual(1);
+    const toDataUrlCall = toDataUrlCalls[0];
+    expect(toDataUrlCall[0]).toEqual("image/jpeg");
+
+    const toCanvasCalls = fileToCanvasMock.mock.calls;
+    expect(toCanvasCalls.length).toEqual(1);
+    const toCanvasCall = toCanvasCalls[0];
+    expect(toCanvasCall[0]).toEqual(mockFile);
+    expect(toCanvasCall[2].maxHeight).toEqual(800);
+    expect(toCanvasCall[2].maxHeight).toEqual(800);
```
Comment (goldpbear) [inline src/base/static/components/form-fields/types/__tests__/add-attachment-button.test.js]: should this be `maxWidth`?

Comment (zmbc) [reply src/base/static/components/form-fields/types/__tests__/add-attachment-button.test.js]: Jesus, good eye. I should use a different method to `expect` on these objects that isn't so brittle.

**Comment by goldpbear** [comment]
Yeah, I think it looks good! Approving...

**Comment by goldpbear** [comment]
I don't why Travis is failing, the tests are passing locally for me.

**Pull request closed without merging**